cubic-server/protocol_id_converter/blockStates.hpp

```diff
@@ -0,0 +1,65 @@
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "nlohmann/json.hpp"
+
+namespace Blocks {
+    // Using int32_t because it's the same size as a varint
+    typedef int32_t BlockId;
```

Comment:
Either use this everywhere and remove `GlobalBlockId` in `types.hpp` or use `GlobalBlockId` here

Reply:
but GlobalBlockId is a uint16_t...
```cpp
typedef uint16_t GlobalBlockId;
```

Reply:
How do I need to change it ?

Reply:
Wut ?

Reply:
wtf, I use this in varInt, this should be int32_t wtf

Reply:
https://github.com/CubicMC/cubic-server/blob/e92ff032f8bb4aa6d99d9cb207a99e04d5ba5d89/cubic-server/types.hpp#L15

Reply:
Do you want me to change that ?

Reply:
Ok no, I add them to the palette and convert them to an `int32_t` so you can just transform them directly